src/Components/AdditionalRepositoriesTable/AdditionalRepositoriesTable.js

```diff
@@ -126,7 +126,7 @@ const AdditionalRepositoriesTable = (props) => {
           })}
         </Tbody>
       </TableComposable>
-      {repositories.length === 0 && <NoAdditionalRepositories />}
+      {repositories?.length === 0 && <NoAdditionalRepositories />}
```

Comment:
Why do we need to make this null safe? Even when there are no additional repositories the response comes back with an empty array. Just the length check should suffice